robot/basestation/app.py

```diff
@@ -89,6 +89,12 @@ def pds():
 def numSections():
     return '4'
 
+@app.route("/stream")
+def stream():
+    """Stream page."""
+    return flask.render_template("pages/Stream.html", roverIP=fetch_ros_master_ip())
+
+# we still using this?
```

Comment:
delete dis

Reply:
done, and the answer is: yes we still need this